doc/whats_new/v0.21.rst

```diff
@@ -286,3 +294,9 @@ These changes mostly affect library developers.
   when `fit` is called twice with the same data, the ouput of
   `predict`, `predict_proba`, `transform`, and `decision_function` does not
   change. :issue:`12328` by :user:`Nicolas Hug <NicolasHug>`
+
+- ``check_estimators_unfitted`` should now raise a ``NotFittedError`` for an
```

Comment:
This is not quite right. "Estimators are now expected to raise a NotFittedError if `predict` or `transform` is called before `fit`; previously an AttributeError or ValueError was acceptable".